core/src/main/resources/defaults/blueprints/cdp-sdx-dev.bp

```diff
@@ -0,0 +1,288 @@
+                    {
+                        "base": true,
+                        "refName": "hdfs-NAMENODE-BASE",
+                        "roleType": "NAMENODE"
+                    },
+                    {
+                        "base": true,
+                        "refName": "hdfs-SECONDARYNAMENODE-BASE",
+                        "roleType": "SECONDARYNAMENODE"
+                    },
+                    {
+                        "base": true,
+                        "refName": "hdfs-DATANODE-BASE",
+                        "roleType": "DATANODE"
+                    },
+                    {
+                        "base": true,
+                        "refName": "hdfs-BALANCER-BASE",
+                        "roleType": "BALANCER"
+                    },
+                    {
+                        "base": true,
+                        "refName": "hdfs-GATEWAY-BASE",
+                        "roleType": "GATEWAY"
+                    },
+                    {
+                        "base": true,
+                        "refName": "hdfs-NFSGATEWAY-BASE",
+                        "roleType": "NFSGATEWAY"
+                    }
+                ],
+                "serviceType": "HDFS"
+            },
+            {
+                "refName": "kafka",
+                "roleConfigGroups": [
+                    {
+                        "base": true,
+                        "refName": "kafka-GATEWAY-BASE",
+                        "roleType": "GATEWAY"
+                    },
+                    {
+                        "base": true,
+                        "refName": "kafka-KAFKA_BROKER-BASE",
+                        "roleType": "KAFKA_BROKER"
+                    }
+                ],
+                "serviceConfigs": [
+                    {
+                        "name": "zookeeper_service",
+                        "ref": "zookeeper"
+                    },
+                    {
+                        "name": "offsets.topic.replication.factor",
+                        "value": "1"
+                    }
+                ],
+                "serviceType": "KAFKA"
+            },
+            {
+                "serviceType": "KNOX",
+                "refName": "knox",
+                "roleConfigGroups": [
+                    {
+                        "base": true,
+                        "refName": "knox-KNOX_GATEWAY-BASE",
+                        "roleType": "KNOX_GATEWAY",
+                        "configs": [
+                            {
+                                "name": "gateway_master_secret",
+                                "value": "cloudera1"
+                            }
+                        ]
+                    },
+                    {
+                        "base": true,
+                        "configs": [
+                            {
+                                "name": "idbroker_master_secret",
+                                "value": "cloudera1"
```

Comment:
Same here.  Please use `{{{ general.password }}}` instead of hard-coding some password.